src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/InheritanceUI.cs

```diff
@@ -26,8 +26,9 @@ public Bitmap InheritanceGlyph
             {
                 if (s_inheritanceGlyph == null)
                 {
-                    s_inheritanceGlyph = new Bitmap(typeof(InheritanceUI), "InheritedGlyph.bmp");
-                    s_inheritanceGlyph.MakeTransparent();
+                    Icon iGlyph = new Icon(typeof(InheritanceUI), "InheritedGlyph");
+                    s_inheritanceGlyph = iGlyph.ToBitmap();
```

Comment:
in other places these two lines are combined 
```suggestion
                    s_inheritanceGlyph = new Icon(typeof(InheritanceUI), "InheritedGlyph").ToBitmap();
```